internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -854,6 +860,28 @@ func (c *ClusterManager) buildObservabilityCloudwatchLoggingCredentialsSecret()
 	}
 }
 
+func (c *ClusterManager) buildObservabilityEnterpriseCloudwatchLoggingCredentialsSecret(orgId string, clusterId string) *k8sCoreV1.Secret {
+	cloudwatchCredentialSecret := &k8sCoreV1.Secret{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: k8sCoreV1.SchemeGroupVersion.String(),
+			Kind:       "Secret",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      c.ObservabilityConfiguration.ObservabilityCloudWatchLoggingConfig.K8sCredentialsSecretName,
+			Namespace: c.ObservabilityConfiguration.ObservabilityCloudWatchLoggingConfig.K8sCredentialsSecretNamespace,
+		},
+		Type: k8sCoreV1.SecretTypeOpaque,
+	}
+
+	stringData := c.ObservabilityConfiguration.ObservabilityCloudWatchLoggingConfig.GetEnterpriseCredentials(orgId)
+	if stringData != nil {
+		cloudwatchCredentialSecret.StringData = stringData
+	}
+
+	glog.Warningf("Failed to find cloudwatch logging secret for cluster %q from organization with ID %q", clusterId, orgId)
```

Comment:
This warning should only be logged when the credentials have not been able to be found

Reply:
This can be rewritten as:
```
if stringData == nil {
  warn
  return secret
}

cloudwatchCredentialSecret.StringData = stringData
return secret
```

or

```
if stringData != nil {
  		cloudwatchCredentialSecret.StringData = stringData
else {
  warn
}
return secret
```